plugins/affiliation/src/main.rs

```diff
@@ -206,113 +221,131 @@ impl AffiliatedType {
 }
 
 /// Returns the number of commits that are flagged for having concerning contributors
-#[query]
-async fn affiliation(engine: &mut PluginEngine, key: Target) -> Result<i64> {
+#[query(default)]
+async fn affiliation(engine: &mut PluginEngine, key: Target) -> Result<Vec<bool>> {
 	log::debug!("running affiliation query");
 
 	// Get the OrgSpec.
-	let org_spec = &CONFIG
-		.get()
-		.ok_or_else(|| {
-			log::error!("tried to access config before set by Hipcheck core!");
-			Error::UnspecifiedQueryState
-		})?
-		.orgs_spec;
+	let org_spec = &ORGSSPEC.get().ok_or_else(|| {
+		log::error!("tried to access config before set by Hipcheck core!");
+		Error::UnspecifiedQueryState
+	})?;
 
 	// Get the commits for the source.
 	let repo = key.local;
-	let value = engine
+	let commits_value = engine
 		.query("mitre/git/commits", repo.clone())
 		.await
 		.map_err(|e| {
 			log::error!("failed to get last commits for affiliation metric: {}", e);
 			Error::UnspecifiedQueryState
 		})?;
-	let commits: Vec<Commit> =
-		serde_json::from_value(value).map_err(|_| Error::UnexpectedPluginQueryInputFormat)?;
+	let commits: Vec<Commit> = serde_json::from_value(commits_value)
+		.map_err(|_| Error::UnexpectedPluginQueryInputFormat)?;
 
 	// Use the OrgSpec to build an Affiliator.
 	let affiliator = Affiliator::from_spec(org_spec).map_err(|e| {
 		log::error!("failed to build affiliation checker from org spec: {}", e);
 		Error::UnspecifiedQueryState
 	})?;
 
-	// Construct a big enough Vec for the affiliation info.
-	let mut affiliations = Vec::with_capacity(commits.len());
+	// Temporary solution to retrieve afilliated commits until batching is implemented
+	// TODO: Once batching works, revert to using looped calls of contributors_to_commit()
 
-	for commit in commits.iter() {
-		// Check if a commit matches the affiliation rules.
-		let hash = commit.hash.clone();
-		let detailed_repo = DetailedGitRepo {
-			local: repo.clone(),
-			details: Some(hash.clone()),
-		};
-		let view_value = engine
-			.query("mitre/git/contributors_for_commit", detailed_repo)
-			.await
-			.map_err(|e| {
-				log::error!("failed to get contributors for commit {}: {}", hash, e);
-				Error::UnspecifiedQueryState
-			})?;
-		let commit_view: CommitContributorView = serde_json::from_value(view_value)
-			.map_err(|_| Error::UnexpectedPluginQueryInputFormat)?;
```

Comment:
Leave this in as a comment block perhaps so the switch back to looping is easy?

Reply:
I was thinking of doing that. Will do.